src/modules/quizzes.js

```diff
@@ -40,16 +40,21 @@ function createQuizUrl(quizId, parameters, options, path) {
   }
 
   if (parameters) {
-    const { section, answers, versionId } = parameters;
+    const { section, answers, quizSessionId, quizVersionId } = parameters;
```

Comment:
Should we add backwards compatibility for `versionId` just in case?

Reply:
https://github.com/Constructor-io/constructorio-client-javascript/pull/245#discussion_r1150840756

@esezen  mentioned that he thought we could delete it. I don't know of anywhere that uses it. What do you think?

Reply:
Hm... from looking at the [logs on s3](https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#logsV2:logs-insights$3FqueryDetail$3D~(end~0~start~-604800~timeType~'RELATIVE~unit~'seconds~editorString~'fields*20*40timestamp*2c*20req.url*2c*20res.statusCode*2c*20responseTime*0a*7c*20filter*20req.url*20*21*3d*20*27*2fhealthcheck*27*0a*7c*20filter*20res.statusCode*20*3d*20200*0a*7c*20filter*20req.url*20like*20*27c*3dciojs-client*27*0a*7c*20filter*20req.url*20like*20*27version_id*27*0a*7c*20sort*20*40timestamp*20desc*0a*7c*20limit*20100~queryId~'7301022b-d3d5-488d-9652-9d9448db53cc~source~(~'ecs-prod-prod-quizzes-main-service))) and applying a filter for `ciojs-client` and `version_id`, I think we're good. I think it should be safe to remove! 👍